ory_mocking/Login/login.go

```diff
@@ -0,0 +1,45 @@
+package Login
+
+import (
+	"encoding/json"
+	testSession "identity_platform_login_ui/ory_mocking/Session"
+	"log"
+	"net/http"
+)
+
+var schemaServerURL string
+
+func SetSchemaServerURL(url string) {
+	schemaServerURL = url
+}
+
+type LoginUpdateResponse struct {
+	Session       testSession.SessionResponse `json:"session"`
+	Session_token string                      `json:"session_token"`
+}
+
+func SelfServiceLoginHandler(w http.ResponseWriter, r *http.Request) {
+	traits := testSession.IdentityTraits{Name: "TestName"}
+	identity := testSession.SessionIdentity{
+		Id:        "test1234",
```

Comment:
Let's not hard code this in the test, other functions may need this value so you would have to hard code the same string to two places which may cause unexpected errors in the future. I think it's best to put it in a global variable, eg:
```go
SESSION_IDENTITY_ID = "test1234"
```

Reply:
I refactored most vars.